src/SqlPersistence/SqlValidateStorageTypeCombinationFeature.cs

```diff
@@ -29,7 +29,8 @@ static void ValidateSagaOutboxCombo(IReadOnlySettings settings)
         }
         var isSagasEnabledForSqlPersistence = settings.IsFeatureActive(typeof(SqlSagaFeature));
         var isOutboxEnabledForSqlPersistence = settings.IsFeatureActive(typeof(SqlOutboxFeature));
-        if (isSagasEnabledForSqlPersistence && isOutboxEnabledForSqlPersistence)
+        if ((isSagasEnabledForSqlPersistence && isOutboxEnabledForSqlPersistence)
+            || (!isSagasEnabledForSqlPersistence && !isOutboxEnabledForSqlPersistence))
```

Comment:
simpler if statement
```suggestion
        if (isSagasEnabledForSqlPersistence == isOutboxEnabledForSqlPersistence)
```

Reply:
I was going to send a PR to simplify this but then I noticed this suggestion, which is exactly what I was intending to propose. Any reason this suggestion wasn't accepted?

Reply:
No reason that I recall apart from "slipped through the cracks"

Reply:
FWIW I sent https://github.com/Particular/NServiceBus.Persistence.Sql/pull/1525 to improve the exception message, and while I was there I boiled down a lot of the class with refactoring.